docs/README.md

```diff
@@ -134,6 +134,19 @@ The override process has two parts:
 1.  Override your template tag with a mock implementation
 2.  Define fake result for your tag in a `yaml` file
 
+
+### Providing a default value for template tags
+To provide a default for a template tag, you need to provide a keyword argument default when overriding your tag.
```

Comment:
At present, this won't work with assigning the ouptut of a tag to a variable like `{% example_tag foo as bar %}` (currently this requires you to set `target_var` in the YAML file). We should document that limitation here or fix it so it works, possibly by checking if the overridden tag is a `simple_tag`?

Reply:
In the interest of getting this functionality into the build I've documented the limitation and we can pickup this extended functionality later.